base_location_nuts/wizard/nuts_import_view.xml

```diff
@@ -36,7 +35,7 @@
 <menuitem action="nuts_import_action"
           id="nuts_import_menu"
           name="Import NUTS 2013"
-          parent="base.menu_localisation"
+          parent="sales_team.menu_localisation"
```

Comment:
This family of modules fall into that menu historically. It's no really needed at all for anything else. There aren't much options on base views anyway... Ideas are welcomed :smile: